src/api.js

```diff
@@ -31,7 +31,7 @@ async function inject(filename, resourceName, resourceData, options) {
   }
   const executableFormat = postject.getExecutableFormat(executable);
 
-  if (!executableFormat) {
+  if (!executableFormat || executableFormat === postject.ExecutableFormat.kUnknown) {
```

Comment:
```suggestion
  if (executableFormat === postject.ExecutableFormat.kUnknown) {
```
I think we could omit the first check because `get_executable_format` always returns one of these values: https://github.com/nodejs/postject/blob/3edd1dd1e0690167d7db0f502620e41888b2f82e/src/postject.cpp#L23-L35